src/contracts/interfaces/IEigenPod.sol

```diff
@@ -82,9 +85,16 @@ interface IEigenPod {
     /// @notice block number of the most recent withdrawal
     function mostRecentWithdrawalBlockNumber() external view returns (uint64);
 
+    /// @notice Returns the validatorInfo struct for the provided pubkeyHash
+    function validatorPubkeyHashToInfo(bytes32 validatorPubkeyHash) external view returns (ValidatorInfo memory);
+
 
     ///@notice mapping that tracks proven partial withdrawals
-    function provenPartialWithdrawal(uint40 validatorIndex, uint64 slot) external view returns (bool);
+    function provenPartialWithdrawal(bytes32 validatorPubkeyHash, uint64 slot) external view returns (bool);
```

Comment:
should this just be `provenWithdrawals` or can there be a partial withdrawal + a full withdrawal for the same `validatorPubkeyHash` in a single `slot`?
I'll look further into what we're doing w/ full withdrawals...